### PR TITLE
Mobile ovs features

### PR DESCRIPTION
**Pull request by nehatdedinca**

### CODE REVIEW

File: jameda_interface_config.js [hunk 1]
```diff
@@ -279,4 +282,42 @@ var interfaceConfig = {
...
+    OS = 'macos';
+} else if (userAgent.match(/Windows/i)) {
+    OS = 'windows';
+}
+
+/**
+ * Checks if the browser is android or ios (mobile).
+ * @returns {?boolean}
+ */
+function isMobileBrowser() {
```
[the remark this answers is not in the document]
Comment (nehatdedinca) [reply jameda_interface_config.js]: Whats the reason for having this complex regex check, is this necessary for our case to know the device besides the OS?

File: jameda_interface_config.js [hunk 2]
```diff
@@ -279,4 +282,42 @@ var interfaceConfig = {
...
+        // 'profile',
+        // 'devices',
+        // 'language'
+        // 'moderator',
+        // 'calendar'
+    ];
+    interfaceConfig.filmStripOnly = true;
+    interfaceConfig.MOBILE_APP_PROMO = false;
+} else {
+    // Do nothing
```
[the remark this answers is not in the document]
Comment (nehatdedinca) [reply jameda_interface_config.js]: Removed

File: jameda_interface_config.js [hunk 3]
```diff
@@ -279,4 +282,42 @@ var interfaceConfig = {
...
+
+/**
+ * Checks if the browser is android or ios (mobile).
+ * @returns {?boolean}
+ */
+function isMobileBrowser() {
+    return OS === 'android' || OS === 'ios';
+}
+
+if (isMobileBrowser()) { // Override features
```
[the remark this answers is not in the document]
Comment (nehatdedinca) [reply jameda_interface_config.js]: Removed

File: jameda_interface_config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nehatdedinca) [reply jameda_interface_config.js]: Removed

File: jameda_interface_config.js [hunk 4]
```diff
@@ -279,4 +282,42 @@ var interfaceConfig = {
...
 
+if (userAgent.match(/Android/i)) {
+    OS = 'android';
+} else if (userAgent.match(/iP(ad|hone|od)/i)) {
+    OS = 'ios';
+} else if (userAgent.match(/Mac(intosh| OS X)/i)) {
+    OS = 'macos';
+} else if (userAgent.match(/Windows/i)) {
+    OS = 'windows';
+}
```
[the remark this answers is not in the document]
Comment (nehatdedinca) [reply jameda_interface_config.js]: Good point, I have refactored the isMobileBrowser function.

File: jameda_interface_config.js [hunk 5]
```diff
@@ -279,4 +279,39 @@ var interfaceConfig = {
...
 };
 
+/**
+ * Checks if the browser is mobile.
+ * @returns {?boolean}
+ */
+function isMobileBrowser() {
+    let check = false;
+
+    (function(a) {
```
Comment (notsaltydev) [inline jameda_interface_config.js]: You don't have to use IIFE. Maybe try to sign to variable result of matched user agent.
e.g.
const regex = ...

function isMobileBrowser() {
    return regex.test(userAgent);
}

Comment (nehatdedinca) [reply jameda_interface_config.js]: There is no need to create global variables which are used only once. Therefore IIFE in this case is useful due to we only need this piece of code to be executed once.

Comment (notsaltydev) [reply jameda_interface_config.js]: Ok, then stay with your solution :)

**Pull request merged**